axon-server/administration/backup-and-messaging-only-nodes.md

```diff
@@ -1,59 +1,132 @@
+
+_Optional parameters_
+
+* _**-S**_ if not supplied connects by default to http://localhost:8024. If supplied, it should be any node serving the _\_admin_ context 
+* _**-t**_  refers to the access token to authenticate at server
+
+C\) Axon Server EE provided REST API \(http:\[server\]:\[port\]/swagger-ui.html\) which offers the _context-rest-controller_ to help perform role maintenance operations
+
 ## Changing node roles <a id="changing-node-roles"></a>
 
 Sometimes you may want to change the role a node has for a specific context. This may happen when you have a pre-existing cluster context configuration \(pre 4.3\) and now you want to be able to start using the new roles. The way to do this is to remove a node from a context and then add it again in the new role.‌
 
-When you remove the node from the context you have an option to preserve the event store. Preserving the event store is recommended when you want to change the role for a node from PRIMARY to BACKUP, or vice versa, as it would prevent a full replication of the event store when the node is added again with the new role. Removing a node while preserving the event store can be done through the UI or through the command line:
+When you remove the node from the context you have an option to preserve the event store. Preserving the event store is recommended when you want to change the role for a node from _PRIMARY_ to _BACKUP_, or vice versa, as it would prevent a full replication of the event store when the node is added again with the new role. 
+
+There are three possible ways to change the role of a node within a context:
+
+A\) The Axon Server EE UI Console. Navigate to the Contexts icon on the navigation menu of the console which will open up the context maintenance screen. You can choose to delete the specific node from the context \(using the delete icon\). In case you would like to preserve the event store, click on the check-box in the pop-up.
+
+B\) The _delete-node-from-context_ command
 
 ```text
-java -jar axonserver-cli.jar delete-node-from-context  -S http://axonserver:8024 -n my-backup-node -c my-context --preserve-event-store
+$ java -jar ./axonserver-cli.jar delete-node-from-context -S http://[node]:[port] -c [context-name] -n [node name]‌
 ```
 
+_Mandatory parameters_
+
+* _**-c**_ refers to an existing context
+* _**-n**_ refers to the node name that should be a member of this context
+
+_Optional parameters_
+
+* _**-S**_ if not supplied connects by default to http://localhost:8024. If supplied, it should be any node serving the _\_admin_ context 
+* _**-t**_  refers to the access token to authenticate at server
+* _**--preserve-event-store**_ removes the node from the context but leaves the event store files on that node.
+
+C\) Axon Server EE provided REST API \(http:\[server\]:\[port\]/swagger-ui.html\) which offers the _context-rest-controller_ to help perform role maintenance operations
```

Comment:
By the way, I've just been notified by @saratry that the `README.md` of Axon Server contains a very nice table depicting what primary, active-backup, passivbe-backup and messaging-only nodes can and cannot do.
Might be nice to include that here as well, as a quick overview.

Reply:
Gotcha, you can find it here: https://github.com/AxonIQ/axon-server#node-types